examples/usage/llava/srt_llava_next_test.py

```diff
@@ -57,10 +57,15 @@ def batch():
 
     mp.set_start_method("spawn", force=True)
     runtime = sgl.Runtime(
-        model_path="lmms-lab/llama3-llava-next-8b",
-        tokenizer_path="lmms-lab/llama3-llava-next-8b-tokenizer",
+        model_path="/mnt/bn/vl-research/checkpoints/onevision/llavanext-google_siglip-so400m-patch14-384-Qwen_Qwen2-7B-Instruct-mid_to_final_next_2p4m_am9_continual_ov",
+        tokenizer_path="lmms-lab/llavanext-qwen-siglip-tokenizer",
+        host="127.0.0.1",
+        tp_size=1,
+        port=8000,
+        chat_template="chatml-llava",
+        disable_flashinfer=True,
```

Comment:
What is the problem with flashinfer? With flashinfer 0.1.4, it should be able to support most models.